api/src/main/java/io/minio/MinioClient.java

```diff
@@ -2987,19 +2987,27 @@ public void removeObject(String bucketName, String objectName)
   /**
    * Removes multiple objects from a bucket.
    *
+   * <p><b>Note:</b></p>
+   * This method does lazy evaluation. You need to traverse over the returned Iterable to do the
+   * real delete jobs.
+   *
    * </p><b>Example:</b><br>
    * <pre>{@code // Create object list for removal.
    * List<String> objectNames = new LinkedList<String>();
    * objectNames.add("my-objectname1");
    * objectNames.add("my-objectname2");
    * objectNames.add("my-objectname3");
+   * // need to traverse over the returned Iterable to do the real delete jobs.
```

Comment:
I don't think this comment is needed as the javadoc is clear to say about lazy eval.